client/src/components/Header.js

```diff
@@ -10,30 +9,36 @@ function Header() {
         <h1 className="u-text--center">
           <a href="/">Murial's Restaurant</a>
         </h1>
-
-        <section>
-          <Link to="/add">AddFood | </Link>
-          <Link to="/details">Details </Link>
-        </section>
+        <div>
```

Comment:
In my head, this div would get styled with something like the following:

```css
position: fixed;
right: 20px;
bottom: 20px;
```

That way the button will "float" over all the content and not take up any space in the top nav.  I'm happy to explore alternatives if this isn't your style.  The main problem I'm trying to solve is the appearance of the main nav.  When the main nav is just a title it elevates the page-level content.  You did a great job with this home page; the images are so impactful and I want to get the menu out of the way.

Reply:
I agree - I think I struggled with floating that button and you did it so easily 💯